code/src/utest/pcgen/cdom/facet/fact/AgeFacetTest.java

```diff
@@ -48,13 +49,14 @@ protected Integer getItem()
 	@Test
 	public void testItemSetMultGetAge()
 	{
-		assertEquals(0, facet.getAge(id));
+
+		assertEquals(0, (int)facet.get(id));
 		getFacet().set(id, 4);
-		assertEquals(4, facet.getAge(id));
+		assertEquals(4, (int)facet.get(id));
```

Comment:
I'm not convinced this is a valuable change, to force a cast vs having type safety beforehand.
This is an architectural decision of type safety, and I'm not convinced this "moves things forward".  This needs further discussion on a list